Snowflake.Data/Core/SFSession.cs

```diff
@@ -185,14 +185,18 @@ internal void renewSession()
             renewSessionRequest.sfRestRequestTimeout = -1;
 
             JObject response = restRequest.post(renewSessionRequest);
-            NullDataResponse sessionRenewResponse = response.ToObject<NullDataResponse>();
+			RenewSessionResponse sessionRenewResponse = response.ToObject<RenewSessionResponse>();
             if (!sessionRenewResponse.success)
             {
                 SnowflakeDbException e = new SnowflakeDbException("", 
                     sessionRenewResponse.code, sessionRenewResponse.message, "");
                 logger.Error("Renew session failed", e);
                 throw e;
-            }
+            } 
+            else 
+            {
+				sessionToken = sessionRenewResponse.data.sessionToken;
+			}

```

Comment:
I think I used space instead of tab and I forgot it's two spaces or four spaces, but please follow the format. Thanks.

Reply:
Will do, thanks.